pyvista/plotting/widgets.py

```diff
@@ -199,6 +203,9 @@ def add_mesh_clip_box(
 
         port = 1 if invert else 0
 
+        if crinkle:
+            mesh.cell_data['cell_ids'] = np.arange(0, mesh.n_cells, dtype=int)
```

Comment:
If this stays (see Matthew's comment) we should simplify it to
```suggestion
            mesh.cell_data['cell_ids'] = np.arange(mesh.n_cells)
```
`dtype=int` is the default if the argument to `arange` is integral.

Reply:
Resolved in e4c9a2ac2b86ffc57822a06df9a3251eb69c639f